demos/imagenet_classification_demo/models.lst

```diff
@@ -0,0 +1,2 @@
+# This file can be used with the --list option of the model downloader.
+alexnet
```

Comment:
Please, extend this list with other models from OMZ.

Reply:
Done

Reply:
Please, sort the list

Reply:
Do you want me to just sort it alphabetically or to divide models into groups by their names (ex.: all mobilenets, then all densenets, ...)?

Reply:
alphabetically

Reply:
Done